pkg/cable/libreswan/libreswan.go

```diff
@@ -460,7 +463,8 @@ func (i *libreswan) serverConnectToEndpoint(connectionName string, endpointInfo
 		"--id", remoteEndpointIdentifier,
 		"--host", "%any",
 		"--client", rightSubnet,
-		"--dpdaction=hold")
+		"--dpdaction=hold",
+		"--dpddelay", strconv.Itoa(dpdDelay))
```

Comment:
Thanks for this PR @ueno , overall code LGTM.
I have short Q:   do we need to set also **dpdtimeout** in case **dpddelay** is set?  

According to https://libreswan.org/man/ipsec.conf.5.html  : 
`dpddelay

Set the delay (in time units, defaults to seconds) between Dead Peer Detection (IKEv1 [RFC 3706](http://www.ietf.org/rfc/rfc3706.txt)) or IKEv2 Liveness keepalives that are sent for this connection (default 0 seconds). Set to enable checking. **If dpddelay is set, dpdtimeout also needs to be se**t.`

Reply:
According to Paul, dpdtimeout is only for IKEv1 and ignored if IKEv2 is used, so if you only use IKEv2 it shouldn't be necessary.

Reply:
correct. with ikev1 message ID is random, and you can send many messages to see if you get a response with the same msgid. For IKEv2, msgid is linear and has a window size. Libreswan uses windows size = 1, so you can only have one IKE message outstanding per IKE SA. So if you send a DPD probe (IKEv2 Informational exchange msg), then you cannot any other IKE message until you get a response. Therefor, not getting a response is fatal, and this is tied to the generic IKE message response timeouts and there is no seperate "DPD timeout"

Reply:
I will fix the man page upstream.